clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -2472,6 +2472,33 @@ bool SemaHLSL::CheckBuiltinFunctionCall(unsigned BuiltinID, CallExpr *TheCall) {
       return true;
     break;
   }
+  case Builtin::BI__builtin_hlsl_dot2add: {
+    // Check number of arguments should be 3
```

Comment:
Nit: imo this comment doesn't really add any new information

Comments should generally be "why" not "what"